### PR TITLE
Upgrade integration to arm64

### PR DESCRIPTION
**Pull request by resdenia**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,35 +91,44 @@ docker run \
...
 ##### Check Logz.io for your logs
 
 Give your logs some time to get from your system to ours,
 and then open [Kibana](https://app.logz.io/#/dashboard/kibana).
 
 ## Versions
 
-0.1.1:
-* Added 'tenant_name' field to the logs
+0.1.2:
```
Comment (mirii1994) [inline README.md:113]: Since we did quite an update to the logstash version, let's bump the version from 0.1.1 to 0.2.0

**Review by mirii1994 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (resdenia) [reply README.md]: Updated in next commit

File: README.md [hunk 2]
```diff
@@ -87,35 +91,44 @@ docker run \
...
+0.2.0:
 
-0.1.0:
-* Sending logs from multiple Okta tenants
-* Sending logs with from every kind of okta domain (not limited to 'okta.com')
-* Note that 'okta_api_key' and 'okta_domain' are now being set in 'tenants-credentials.yml' and no longer as environment parameters.
+- Changed logstash.conf routing
+- Updated version logstash version to `docker.elastic.co/logstash/logstash:7.16.1`
+- Changed yaml handler from `ruamel.yaml` to `pyyaml`
+- Implemented `pyyaml` to the integration
```
Comment (mirii1994) [inline README.md:118]:
```suggestion
- Changed logstash.conf routing.
- Upgraded Logstash version 7.16.1.
- Changed yaml handler from `ruamel.yaml` to `pyyaml`.
- Added ARM support.
- Upgraded image to use python3.
```

**Review by mirii1994 — CHANGES_REQUESTED** [review]
See my notes.

**Review by mirii1994 — APPROVED** [review]
Great work!

**Pull request merged**